cognite/seismic/protos/query_service_messages.proto

```diff
@@ -200,6 +217,15 @@ message PositionQuery {
     int32 xline = 2;
 }
 
+/**
+Point in volume defined by its inline, crossline and time
+**/
+message PositionInVolume {
+    int32 iline = 1;
+    int32 xline = 2;
+    int32 time = 3;
```

Comment:
Should we call it z, so that it can mean either depth or time? Also, the request from Cegal was not to have index (int32), but an actual depth or time range, but maybe we could have that as a second stage of the feature - it would require using the headers to get the distance between samples and convert indices to real values of time/depth

Reply:
done